test/index.spec.js

```diff
@@ -21,10 +21,16 @@ const asyncActionsCreator = (actions, timeout = 10) => (dispatch) => {
 };
 const actionsCreator = (actions) => (dispatch) => actions.forEach((action) => dispatch(action));
 const createMockStore = (middlewares = []) => configureStore(middlewares)();
+const originalSetTimeout = setTimeout;
+const originalClearTimeout = clearTimeout;
 
 afterEach(() => {
     jest.useRealTimers();
     jest.clearAllMocks();
+    Object.assign(global, {
```

Comment:
If you adhere to my suggestion, we can remove this and simply replace `clearAllMocks` with `restoreAllMocks()`

Reply:
👍